### PR TITLE
Collapsed Reply Threads General Availability

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation updates for:
- https://github.com/mattermost/mattermost-server/pull/20114
- https://github.com/mattermost/mattermost-webapp/pull/9404
- https://github.com/mattermost/mattermost-webapp/pull/10465
- https://github.com/mattermost/mattermost-server/pull/20088

Updated end user and administrative Collapsed Reply Threads product documentation to:
- remove beta labels
- note that CRT is enabled by default for all users in all new Mattermost deployments

@rmonga71 - Can you provide me with a link to the updated YouTube training video when it's available for inclusion in the product documentation, please?

@esethna:
- Existing screenshots are using latest branding UI. Please let me know if you'd like to see any of the existing screenshots updated.
- Placeholders are included for the new KB article that we'll be working on together.
- Incorporating mobile v2 (vs web/desktop) workflows is work we plan on taking on a bit later this year. Any concerns about holding off on adding these workflows as part of v7.0?

### CODE REVIEW

**Comment by esethna** [comment]
Thanks Carrie!


> Incorporating mobile v2 (vs web/desktop) workflows is work we plan on taking on a bit later this year. Any concerns about holding off on adding these workflows as part of v7.0?

No concerns on holding off on mobile for now. The UI/UX is very similar

**Comment by rmonga71** [comment]
Thanks Carrie - will do re: the tutorial video. Aiming to pass to marketing by end of week for mastering.

File: source/channels/organize-conversations.rst
Comment (esethna) [inline source/channels/organize-conversations.rst:75]: New vid link coming from @rmonga71

File: source/configure/configuration-settings.rst
Comment (esethna) [inline source/configure/configuration-settings.rst:2213]: "Customers upgrading to v7.0 must review the" > "Administrators should review the"

Comment (esethna) [inline source/configure/configuration-settings.rst:2213]: "prior to enabling this functionality" > "prior to enabling this functionality in self-managed deployments"

File: source/configure/configuration-settings.rst
Comment (esethna) [inline source/configure/configuration-settings.rst:2209]: Move end user docs link to this paragraph and remove it from the important note

File: source/configure/configuration-settings.rst
Comment (esethna) [inline source/configure/configuration-settings.rst:2217]: Mattermost Cloud workspaces have Collapsed Reply Threads set to ``always_on`` by default.

Comment (esethna) [inline source/configure/configuration-settings.rst:2217]: This is the recommended configuration for optimal user experience and to ensure consistency in how users read and respond to threaded conversations

File: source/configure/configuration-settings.rst
Comment (esethna) [inline source/configure/configuration-settings.rst:2223]: Remove this part "To disable all backend operations and thread tracking for Collapsed Reply Threads, you must also disable the `ThreadAutoFollow <https://docs.mattermost.com/configure/configuration-settings.html#automatically-follow-threads>`__ configuration setting."

Since we don't want admins disabling this unless absolutely necessary

**Review by esethna — CHANGES_REQUESTED** [review]

**Comment by esethna** [comment]
Another doc with reference to CRT beta: https://docs.mattermost.com/channels/channels-settings.html

File: source/configure/configuration-settings.rst
Comment (cwarnermm) [inline source/configure/configuration-settings.rst:2190]:
```suggestion
This setting must be enabled to support `Collapsed Reply Threads <https://docs.mattermost.com/channels/organize-conversations.html>`__. See the `administrator’s guide to enabling Collapsed Reply Threads <https://mattermost.zendesk.com/knowledge/articles/6880701948564/en-us?brand_id=327479>`__ knowledge base article for details.
```

File: source/configure/configuration-settings.rst
Comment (cwarnermm) [inline source/configure/configuration-settings.rst:2213]:
```suggestion
    Administrators upgrading to v7.0 must review the `administrator’s guide to enabling Collapsed Reply Threads <https://mattermost.zendesk.com/knowledge/articles/6880701948564/en-us?brand_id=327479>`__ knowledge base article to learn about the system requirements, steps to enable, and self-host prerequisites to consider prior to enabling this functionality. See our `Organizing Conversations using Collapsed Reply Threads <https://docs.mattermost.com/channels/organize-conversations.html>`__ documentation to learn more about this feature.
```

File: source/configure/configuration-settings.rst
Comment (cwarnermm) [inline source/configure/configuration-settings.rst:2217]:
```suggestion
**Always On**: Enables Collapsed Reply Threads functionality on the server and for all users. Users can't disable this functionality. Mattermost Cloud workspaces have Collapsed Reply Threads set to ``always_on`` by default.
```

File: source/configure/configuration-settings.rst
Comment (cwarnermm) [inline source/configure/configuration-settings.rst:2223]:
```suggestion
**Disabled**: Disables Collapsed Reply Threads front-end functionality.
```

**Comment by esethna** [comment]
@cwarnermm Matt updated the screenshots in this Figma, you can grab new screenshots here: https://www.figma.com/file/9h1lwOzlBAhRQJYojJ73KR/MM-44996-CRT-Updated-Marketing-Screenshots?node-id=2%3A33292 

For the youtube vid, we can merge and update it once available

File: source/channels/channels-settings.rst
Comment (justinegeffen) [inline source/channels/channels-settings.rst:184]:
```suggestion
Depending on how your System Admin has enabled **Collapsed Reply Threads** for your workspace, it may already be enabled for you, or you may be able to enable this feature for your account. See our `organize conversations using Collapsed Reply Threads <https://docs.mattermost.com/channels/organize-conversations.html>`__ documentation to learn more about working with Collapsed Reply Threads.
```

File: source/channels/organize-conversations.rst
Comment (justinegeffen) [inline source/channels/organize-conversations.rst:87]:
```suggestion
See our `configuration settings <https://docs.mattermost.com/configure/configuration-settings.html#collapsed-reply-threads>`__ documentation for details on configuring the default availability of collapsed reply threads. From Mattermost v7.0, Collapsed Reply Threads are enabled by default for all new Mattermost deployments.
```

File: source/channels/organize-conversations.rst
Comment (justinegeffen) [inline source/channels/organize-conversations.rst:89]: Not sure if this line needed the formatting that I deleted in my previous comment but noting it here in case it did!

**Review by justinegeffen — APPROVED** [review]
Three comments, none blocking, so approving in the meantime! Awesome job!!

**Comment by cwarnermm** [comment]
@esethna - Any final changes you'd like to see made to the Collapsed Reply Threads documentation for v7.0?

**Review by esethna — APPROVED** [review]
Thanks Carrie!!

File: source/channels/organize-conversations.rst
Comment (rmonga71) [reply source/channels/organize-conversations.rst:76]: Thanks Carrie. We should have it by EOD tomorrow (Tuesday)

**Pull request merged**
